content/01.abstract.md

```diff
@@ -1,3 +1,14 @@
-## Abstract {.page_break_before}
-
-
+## Abstract {.page_break_before} 
+Since late 2019, Coronavirus disease 2019 (COVID-19) has spread around the world, resulting in the
```

Comment:
I would recommend making this (and other sections - intro and the things you aim to fill in) one sentence per line instead of fixed line width. We tried fixed line width on https://github.com/greenelab/deep-review and it was a nightmare to maintain. One sentence per line ended up working well with comments.

Reply:
Should be fixed [here](https://github.com/greenelab/covid19-review/pull/1/commits/be27fbb90ef9f51987a03da978c2cdf6bc732926)